pkg/network/ebpf/c/prebuilt/tracer.c

```diff
@@ -591,7 +591,8 @@ int kprobe__sockfd_lookup_light(struct pt_regs* ctx) {
     };
     struct sock** sock = bpf_map_lookup_elem(&sock_by_pid_fd, &key);
     if (sock != NULL) {
-        return 0;
```

Comment:
This isn't needed anymore, since the stale sock struct bug was fixed with https://github.com/DataDog/datadog-agent/pull/9462/commits/868b21fe96cf4e8e27cadeac1dbc24be805ef857